recipe/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "pypdfium2" %}
+{% set version = "4.18.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pypdfium2-team/{{ name }}/releases/download/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c937121dc475942697fbb3e04ffa7b28d36afc2b76cc9aac22fbd327c6dc6d61
+  patches:
+    - 0001-use-pdfium-library-from-repack-dependency.patch
+    - 0002-copy-binding-file-from-ctypesgen-output-path.patch
+    - 0003-fix-windows-include-location.patch
+
+build:
+  skip: True  # [linux and (ppc64le or s390x)]
+  entry_points:
+    - pypdfium2 = pypdfium2.__main__:cli_main
+  script:
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - git         # [not win]
+    - {{ compiler('c') }}
+    - patch       # [not win]
+    - m2-patch    # [win]
+  host:
+    - python
+    - setuptools
+    - pip
+    - build
```

Comment:
I have a feeling this isn't required.

Reply:
I think you're right. `build` isn't required here. This error was inherited from upstream, so my fault basically. 😅
IIRC `build` is only called as subprocess in `craft_packages.py`, which runs only in our workflow, not on user setup. So it doesn't belong in the pyproject build-system requires.